test/extensions/common/wasm/test_data/test_cpp/bazel/external/proxy-wasm-cpp-sdk.BUILD

```diff
@@ -0,0 +1,34 @@
+licenses(["notice"])  # Apache 2
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "proxy_wasm_intrinsics",
+    srcs = [
+        "proxy_wasm_intrinsics.cc",
+        "proxy_wasm_intrinsics_lite.pb.cc",
+        "struct_lite.pb.cc",
+    ],
+    hdrs = [
+        "proxy_wasm_api.h",
+        "proxy_wasm_common.h",
+        "proxy_wasm_enums.h",
+        "proxy_wasm_externs.h",
+        "proxy_wasm_intrinsics.h",
+        "proxy_wasm_intrinsics.pb.h",
+        "proxy_wasm_intrinsics_lite.pb.h",
+        "struct_lite.pb.h",
+    ],
+    visibility = ["//visibility:public"],
+    deps = [
+        "@com_google_protobuf//:protobuf_lite",
+    ],
+)
+
+filegroup(
+    name = "jslib",
+    srcs = [
+        "proxy_wasm_intrinsics.js",
+    ],
+    visibility = ["//visibility:public"],
+)
```

Comment:
This should be part of `proxy-wasm-cpp-sdk`'s `BUILD` file and only loaded here.

Reply:
Done